decide/src/main/java/org/example/InitialSettings.java

```diff
@@ -22,13 +31,36 @@ public class Parameters {
         public final int D_PTS; // Number of intervening points in LIC 9
         public final int E_PTS; // Number of intervening points in LIC 10, 14
         public final int F_PTS; // Number of intervening points in LIC 10, 14
-        public final int G_PTS;
-        public final double LENGTH2;
-        public final double RADIUS2;
-        public final double AREA2;
+        public final int G_PTS; // Number of intervening points in LIC 11
+        public final double LENGTH2; // Maximum length in LIC 12
+        public final double RADIUS2; // Maximum radius in LIC 13
+        public final double AREA2; // Maximum area in LIC 14
+
+
+        public Parameters(double length1, double radius1, double epsilon, double area1, int q_pts, int quads, double dist, int n_pts, int k_pts, int a_pts, int b_pts, int c_pts, int d_pts, int e_pts, int f_pts, int g_pts, double length2, double radius2, double area2) {
+            LENGTH1 = length1;
+            RADIUS1 = radius1;
+            EPSILON = epsilon;
+            AREA1 = area1;
+            Q_PTS = q_pts;
+            QUADS = quads;
+            DIST = dist;
+            N_PTS = n_pts;
+            K_PTS = k_pts;
+            A_PTS = a_pts;
+            B_PTS = b_pts;
+            C_PTS = c_pts;
+            D_PTS = d_pts;
+            E_PTS = e_pts;
+            F_PTS = f_pts;
+            G_PTS = g_pts;
+            LENGTH2 = length2;
+            RADIUS2 = radius2;
+            AREA2 = area2;
+        }
     }
 
-    public class Point {
+    public static class Point {
```

Comment:
Why do we make `Point` static?

Reply:
Oh yes you're right, that's a mistake I'll change it.